homeassistant/components/blebox/sensor.py

```diff
@@ -0,0 +1,33 @@
+"""BleBox sensor entities."""
+
+from homeassistant.helpers.entity import Entity
+
+from . import BleBoxEntity, create_blebox_entities
+from .const import BLEBOX_DEV_CLASS_MAP, BLEBOX_TO_UNIT_MAP, DOMAIN, PRODUCT
+
+
+async def async_setup_entry(hass, config_entry, async_add):
+    """Set up a BleBox entry."""
+
+    product = hass.data[DOMAIN][config_entry.entry_id][PRODUCT]
+    create_blebox_entities(product, async_add, BleBoxSensorEntity, "sensors")
+    return True
```

Comment:
Nothing is checking this return value. We can remove the statement.

Reply:
I believe it's checked here: https://github.com/home-assistant/core/blob/b4404b071f375bf29812036108304c2017b1679b/homeassistant/config_entries.py#L253

And it's expected to be a bool: https://github.com/home-assistant/core/blob/b4404b071f375bf29812036108304c2017b1679b/homeassistant/config_entries.py#L219

Let me know if I'm wrong.

Reply:
No, that's the function in the component module, ie the init module in the integration.

Reply:
Got it.

I refactored further and decided to fix this separately for your convenience (and to make sure it's applied to all other upcoming BleBox platform PRs): https://github.com/home-assistant/core/pull/35552

Let me know if you want this merged into existing platform-specific PRs.